.rubocop.yml

```diff
@@ -0,0 +1,54 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'Gemfile'
+    - '**/*.rake'
+    - 'bin/**/*'
+    - 'config/**/*'
+    - 'db/**/*'
+    - 'node_modules/**/*'
+    - 'vendor/**/*'
+    - 'tmp/**/*'
+    - 'Rakefile'
+    - 'config.rb'
+    - 'config.ru'
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
```

Comment:
The default sounds good to me.

I think double-quote usage should be reserved for when it means something, so as to draw attention to whatever advanced/interpolation stuff is going on in our strings.

Reply:
This Pull Request is already rather large, so I personally think it would be okay to include the quotation mark normalization here as well. If it is committed in a neat way here, that would make it easy to review.

However, maybe saving it for a separate Pull Request is good for this reason: We have "squash pull requests" enabled at this repository. So this entire Pull Request is destined to become one commit on the `develop` branch. Perhaps we should want the quotation mark normalization done separately so it stays a separate commit on `develop` branch.

Reply:
Again, thanks for the thoughtful approach to enabling this linter. Anticipating some of the pitfalls, however minor, increases my confidence with adding the linter and is making the experience better of adding it to the repository. It's personally my first time using it, so that's great.